source/releasenotes/about.rst

```diff
@@ -17,71 +17,30 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a 4.20 LTS minor release with over 150 fixes
-and improvements since the 4.20.0.0 release. Some of the highlights include:
-
-• Improvements to multi-architecture support in CloudStack
-• vTPM support for KVM and VMware
-• Support for XenServer 8.4 / XCP-ng 8.3
-• Added support for VMware 80u2 and 80u3
-• Updated Sysyem VM template to Debian 12.11
-• NAS B&R improvements
-• Experimental Support of EL10 as Management Server and KVM host
+Apache CloudStack |release| is a 4.21 Regular release with 15 new features,
+around 40 improvements and more than 150 bug fixes since the 4.20.1.0 release.
+Some of the highlights include:
+
+• CloudStack Extensions Framework and Orchestrate Anything extension, with built-in extensions for Proxmox and Hyper-V
+• GPU Integration with KVM in CloudStack (Technical Preview)
+• Netris Network Plugin Integration
+• Cloudian HyperStore Object Store Plugin Integration
+• CloudStack Kubernetes Service (CKS) Enhancements
+• Create Instance from Backup
+• Instance Lease (Automatic Stop/Deletion)
+• Backup and Object Storage Limits
+• Virtual Trusted Platform Module (vTPM) Support
+• Support for KVM on s390x (IBM Z and LinuxONE) architecture
```

Comment:
Not sure about highlighting this but I don't know of compatibility

Reply:
agree, PR is in & seems tested